docs/siem/detections/detections-ui-exceptions.asciidoc

```diff
@@ -0,0 +1,228 @@
+
+. Select the list type (`Keywords`, `IP addresses`, `IP ranges`, or
+`Text`)
+. Drag or select the `csv` or `txt` file that contains the values.
+. Click *Upload list*.
+
+NOTE: When the name of the file you are uploading already exists, the values in
+the new file are appended to the previously uploaded values.
+
+To view, delete, or export existing lists:
+
+. Go to *Security* -> *Detections* -> *Manage detection rules*.
+. In the *Value lists* pane, click the required action icon.
+
+[float]
+[[detection-rule-exceptions]]
+=== Add detection exceptions to a rule
+
+You can add exceptions to a rule via the Rule details page or the Alerts table.
+When you add an exception, you can also close all alerts that meet the
+exception's criteria.
+
+IMPORTANT: When you select to close all alerts that meet the exception's
+criteria, all matching alerts are closed, *including* alerts generated by other
+rules.
+
+. To add an exception via the Rule details page:
+.. Go to the Rule details page of the rule to which you want to add the
+exception (*Security* -> *Detections* -> *Manage detection rules* ->
+<rule name>).
+.. Scroll down to the *Trend* histogram and select the *Exceptions* tab.
+.. Click *Add new exception*.
+. To add an exception via the Alerts table:
+.. Go to Detections (*Security* -> *Detections*).
+.. Scroll down to the Alerts table and click the more actions icon, and then
+select *Add exception*.
++
+The *Add Exception* window opens (via Alerts table).
++
+[role="screenshot"]
+image::images/add-exception-ui.png[]
+
+. Add conditions that define when the exception prevents alerts. You can define
+multiple conditions with `OR` and `AND` relationships. In the example above,
+the exception prevents the rule from generating alerts when the
+`maintenance.exe` process runs on `win-server-1`, `win-server-2`, or
+`win-server-3`.
++
+[IMPORTANT]
+============
+You can use nested conditions. However, this is only required for
+<<nested-field-list, these fields>>. For all other fields, nested conditions
+should not be used. 
+============
++
+If you have created value lists, you can use them to exclude or include all
+values in a list with `is in list` and `is not in list` operators:
++
+[role="screenshot"]
+image::images/exceptions-ui-list.png[]
+
+NOTE: When using a list, all exception statements must use `is in list` and
+`is not in list` operators.
+
+. You can select any of the following:
+
+* _Close this alert_: Closes the alert when the exception is added. This option
+is only available when adding exceptions via the Alerts table.
+* _Close all alerts that match this exception, including alerts generated by other rules_:
+Closes all alerts that match the exception's conditions.
+
+. Click *Add Exception*.
+
+[float]
+[[endpoint-rule-exceptions]]
+=== Add Elastic Endpoint Security exceptions
+
+Like detection rule exceptions, you can add Endpoint agent exceptions via both
+the Elastic Endpoint Security rule and its generated alerts. 
```

Comment:
This might be a good spot to mention the warning/info segment from my above comment:


> 2. The Detection Alert generated from a rule has the following fields: 
`signal.original_event.module determined:endpoint and signal.original_event.kind:alert `, in which case the `Add endpoint exception` action [will be available](https://github.com/spong/kibana/blob/c21474b4ce029b820072b53f3908075404bccbde/x-pack/plugins/security_solution/public/detections/components/alerts_table/default_config.tsx#L310-L320), **and if used will end up associating the rule that generated the alert with the `Global Endpoint Exception List`.** 
> 
> We don't have copy for this latter part in the app, so we should call this out somewhere in the documentation (not sure if here works, but might fit well?).

Reply:
Also added the first point to create rule Elastic endpoint exceptions option.